packages/client/tsconfig.noImplicitAny.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "noImplicitAny": true,
```

Comment:
+1 Do we still need it? or will it be removed in part 2?

Reply:
removed in part 2!